troubadix/standalone_plugins/deprecate_vts.py

```diff
@@ -55,10 +71,10 @@ def update_summary(file: DeprecatedFile, deprecation_reason: str) -> str:
         print(f"No summary in: {file.name}")
         return file.content
 
-    deprecate_text = (
-        f"Note: This VT has been deprecated "
-        f"{Deprecations[deprecation_reason].value}"
-    )
+    deprecate_text = f"Note: This VT has been deprecated {Deprecations[deprecation_reason].value}"
```

Comment:
I would suggest something like e.g. this:

```
deprecate_text = f"\n  Note: This VT has been deprecated {Deprecations[deprecation_reason].value}"
```

to have a "final" text like e.g.:

```
  script_tag(name:"summary", value:"The remote host is missing an update for the 'perl' package(s) advisory.

  Note: This VT has been deprecated as a duplicate. The replacement VT has OID 1.3.6.1.4.1.25623.1.1.18.1.2024.0021.1.");
```

otherwise this will look like a complete single line within the GSA due to the lack of extended formatting support within it.